apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -252,6 +255,124 @@ const CustomizableCurriculumCatalogCard = ({
     }
   };
 
+  if (wide) {
```

Comment:
Are there any HTML structural differences to what is returned in the `wide` version other than styling? Just wondering if there's a way to pass in or use different styles rather than having a duplicate layout of the card in case we want to make future structural changes to the cards. This is for sure not blocking and just more of a thought, totally fine proceeding with this as well since the cards are being used in different places.